### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by carsonwright**

### CODE REVIEW

File: src/lib/chart-override.js [hunk 1]
```diff
@@ -0,0 +1,354 @@
+import Chart from "chart.js";
```
Comment (tomprats) [inline src/lib/chart-override.js]: I think this is old code

File: test/tests/slidedeck.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/*global sinon,expect,StepTest*/
+/*eslint-disable prefer-arrow-callback*/
+function Init(){
+  this.test("Slidedeck Should Initialize")
```
Comment (tomprats) [inline test/tests/slidedeck.js]: To be consistent, we should either always use SlideDeck/slide-deck or Slidedeck/slidedeck. In classes/folders

File: src/lib/traitify.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
...
-Traitify.ui.client = Traitify;
-Traitify.__version__ = __VERSION__;
+
+/*global __VERSION__ */
+import TraitifyLib from "traitify-client";
 require("polyfills");
-export default Traitify;
\ No newline at end of file
+import ui from "traitify-ui";
+function Init(){
```
Comment (tomprats) [inline src/lib/traitify.js]: This seems kinda odd to me. What's the reasoning behind it?

Comment (carsonwright) [inline src/lib/traitify.js]: The reasoning to include an init, instead of allowing code to run instantly?

Comment (tomprats) [inline src/lib/traitify.js]: Looks like in another place you export the Init function, but here you export the the result of the Init function. What's the purpose of either? I don't have much experience around this so I just wanted to know

Comment (carsonwright) [inline src/lib/traitify.js]: The reason for requiring the init is that you then can force webpack to make a new copy of the required return every time its required

i.e. if you just return an object webpack will share required modules, but if you put it in a function and then call that function and within that scope set Traitify to extend TraitifyLib then you get a new version of the TraitifyLib every time

Comment (carsonwright) [inline src/lib/traitify.js]: In other places I need to be able to init without requiring it again, but here I can just wrap it with this.

Comment (carsonwright) [inline src/lib/traitify.js]: There may be better ways to do it, I'd be down for a refactor after this, would love to just see this code out there though

File: src/lib/traitify-client.js [hunk 4]
```diff
@@ -1,36 +1,43 @@
...
-Traitify.host =  "http://api.traitify.com";
+  static setPublicKey(key){
+    this.publicKey = key;
+    TraitifyLib.setPublicKey(key);
+  }
+  static setHost (host){
+    this.host = host;
+  }
+}
+TraitifyLib.host = "";
```
Comment (carsonwright) [inline src/lib/traitify-client.js]: This was needed so it could be mocked

File: src/lib/traitify-ui.js [hunk 5]
```diff
@@ -66,6 +67,7 @@ export default class TraitifyUI {
           return reject("Your target element could either not be selected or was not provided");
         }
 
+        ceaou;
```
Comment (carsonwright) [inline src/lib/traitify-ui.js]: This was an error, but was taken out in another pull request that came after this one.

Comment (tomprats) [inline src/lib/traitify-ui.js]: :+1:

**Pull request merged**